scripts/release_scripts/update_changelog_and_credits.py

```diff
@@ -489,29 +489,35 @@ def main():
             release_constants.RELEASE_SUMMARY_FILEPATH))
 
     common.ask_user_to_confirm(
-        'Check emails and names for authors and contributors in the release '
-        'summary file and verify that the emails are '
+        'Check emails and names for new authors and new contributors in the '
+        'file: %s and verify that the emails are '
         'correct through welcome emails sent from welcome@oppia.org '
```

Comment:
Do we expect all the release team members to receive welcome email?

Reply:
Yes, everyone who fills a CLA gets a welcome email and it is compulsory to fill a CLA to submit a PR.